net/goeap_proxy/files/etc/init.d/goeap_proxy

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh /etc/rc.common
+#
+
+START=25
+USE_PROCD=1
+
+PROG="/usr/bin/goeap_proxy"
+
+boot()
+{
+    config_load goeap_proxy # load config files
+    config_get wan devices wan # create a variable var; letting it know it's a device and assigning the 'wan'
+    config_get lan devices lan
+    ubus -t 30 wait_for "network.${wan}.device" "network.${lan}.device" # updated for DSA (v22.02+)
+    rc_procd start_service
+}
+
+start_service()
+{
+    config_load goeap_proxy # load config files
+    config_get wan devices wan
+    config_get lan devices lan4 # change 'lan4' to your switch's port
```

Comment:
Would be cleaner to fail if config is missing, lan4 is not a viable default

Reply:
Let me read up on nftables!